### DISCUSSION TITLE
Self-Nominate for the role of Maintainer

### DISCUSSION THREAD

**Discussion by TisVictress** [paketo-buildpacks/feedback#176] [Web Servers Team]
I'd like to nominate myself to become a maintainer for web-servers. [Here](https://github.com/paketo-buildpacks/web-servers/commit/47a73cf2fdfd443874c908650ba6d9abcaff141c) was my latest contribution to web-servers.

@paketo-buildpacks/web-servers-maintainers 👍 / 👎 ?

**Comment by TisVictress** [comment]
CC @paketo-buildpacks/steering-committee

**Comment by dmikusa** [comment]
Disregard my last note, I think I was looking at the wrong team or something.

Anyway, we have the votes here but normally for maintainer roles the standard process is to have [an existing maintainer nominate you, no self-nominations](https://github.com/paketo-buildpacks/community/blob/main/GOVERNANCE.md#maintainers). So that we can follow protocol, @paketo-buildpacks/web-servers-maintainers can one of you please respond and officially nominate @TisVictress. Thanks

**Comment by ForestEckhardt** [comment]
@dmikusa-pivotal I will happily nominate @TisVictress.
Would this message suffice or would you prefer a brand new discussion thread?

**Comment by dmikusa** [comment]
Thanks @ForestEckhardt and sorry all for the additional protocol requirements. We just want to be as transparent and fair to all as possible, so I have to follow policy to the letter. @TisVictress - Congrats! You're now on the maintaineirs team, I'll send the update through to the paketo community TEAMS.md doc in a second.
